src/main/webapp/login/login.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
+import FormControl from '@material-ui/core/FormControl'
+import InputAdornment from '@material-ui/core/InputAdornment'
+import Visibility from '@material-ui/icons/Visibility'
+import VisibilityOff from '@material-ui/icons/VisibilityOff'
+import FormLabel from '@material-ui/core/FormLabel'
+import OutlinedInput from '@material-ui/core/OutlinedInput'
+import InputLabel from '@material-ui/core/InputLabel'
+import IconButton from '@material-ui/core/IconButton'
+
+export const LogIn = props => {
+  const [values, setValues] = React.useState({
+    username: '',
+    password: '',
+    showPassword: false,
+  })
+  const handleChange = prop => event => {
+    setValues({ values, [prop]: event.target.value })
+  }
+  const changePasswordVisibility = () => {
+    let flip = !values['showPassword']
+    setValues({ showPassword: flip })
+  }
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100%',
+        maxHeight: '100%',
+        padding: 20,
+        boxSizing: 'border-box',
+        ...props.style,
+      }}
+    >
+      <FormLabel style={{ padding: 10 }}>{props.label}</FormLabel>
+      <TextField
+        autoFocus
+        required
+        fullWidth
+        variant="outlined"
+        label="Username"
+        style={{ marginBottom: 20 }}
+        onBlur={handleChange('username')}
+      />
+      <FormControl
+        fullWidth
+        required
+        variant="outlined"
+        style={{ marginBottom: 20 }}
+      >
+        <InputLabel>Password</InputLabel>
+        <OutlinedInput
```

Comment:
Why is this not a TextField component?

Reply:
The text field component doesn't support the `EndAdornment` Prop which I added to let the user toggle the visibility of their password in the modal